addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1221,7 +1228,7 @@ MockServer.include({
     _mockMailNotification_NotificationFormat(ids) {
         const notifications = this._getRecords('mail.notification', [['id', 'in', ids]]);
         return notifications.map(notification => {
-            const partner = this._getRecords('res.partner', [['id', '=', notification.res_partner_id]])[0];
+            const partner = this._getRecords('res.partner', [['id', '=', notification.res_partner_id[0]]])[0];
```

Comment:
This doesn't look correct, `_getRecords` is returning the object in this.data (so the ID for res_partner_id), not the result of read (which would be [id, display_name]), unless I'm missing something?

Reply:
It's what is done server-side.
```python
    def _notification_format(self):
        """Returns the current notifications in the format expected by the web
        client."""
        return [{
            'id': notif.id,
            'notification_type': notif.notification_type,
            'notification_status': notif.notification_status,
            'failure_type': notif.failure_type,
            'res_partner_id': [notif.res_partner_id.id, notif.res_partner_id.display_name],
        } for notif in self]
```